alma/alma-woocommerce-gateway
# Integrate credit on in page

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-2072/[-wc]-integrate-credit-for-in-page)

### Code changes


We add a new gataway for inpage on credit.

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Run unit tests locally.
Connect to the [QA env](https://qa-inpage-credit-woocommerce-8-2-2.integrations.getalma.eu/), check if inpage is activated, add a product to the cart, go to the checkout and try to pay with the p10x.

### Checklist for authors and reviewers




### Non applicable
